tutorials/introductory/usage.py

```diff
@@ -149,14 +149,15 @@
 # Matplotlib, pyplot and pylab: how are they related?
 # ====================================================
 #
-# Matplotlib is the whole package; :mod:`matplotlib.pyplot`
-# is a module in matplotlib; and :mod:`pylab` is a module
-# that gets installed alongside :mod:`matplotlib`.
+# Matplotlib is the whole package and :mod:`matplotlib.pyplot` is a module in
+# Matplotlib.
```

Comment:
Either remove pylab from the headline as well or say something about it in the text. :smile:

Reply:
it does appear, just later in the section.